eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,12 +1,38 @@
+resources:
+  repositories:
+    - repository: azure-sdk-tools
```

Comment:
As a follow-up item for all the language repo's please figure out all the places we are using these repository references and see if we can eliminate them. That goes for both azure-sdk-tools and azure-sdk-build-tools. I suspect azure-sdk-build-tools should only ever be needed for java and .NET.